src/Selenium2Library/keywords/_browsermanagement.py

```diff
@@ -507,6 +507,34 @@ def set_browser_implicit_wait(self, seconds):
         implicit_wait_in_secs = robot.utils.timestr_to_secs(seconds)
         self._current_browser().implicitly_wait(implicit_wait_in_secs)
 
+    def get_browser_info(self, type='object'):
+        """Gets information from webdriver on current browser.
+
+        By default returns an Object, otherwise if the argument `type`is different of 'object', returns
+        a Dictionary.
+
```

Comment:
Perhaps it would be good to refer the Selenium API documentation too: http://selenium-python.readthedocs.org/en/latest/api.html?highlight=desired_capabilities#selenium.webdriver.remote.webdriver.WebDriver.desired_capabilities